_posts/2021-12-07-g2p-case-feeding.md

```diff
@@ -40,7 +42,7 @@ We assume:
 
 To reproduce the examples below, you will need to install `g2p` on your own machine by following the instructions at [GitHub/g2p](https://github.com/roedoejet/g2p) and make sure the `g2p convert` command line works.
 
-You'll need a recent version of `g2p`, since which supports the `--config` option is a new feature. Release v0.5.202112?? or more recent will work. (TODO create that release before this blog post is published, since I depend on commits made 2021-12-02. See PR #143 TODO)
+You'll need a recent version of `g2p`, since we'll make use of the new `--config` option to `g2p convert`. Release v0.5.202112?? or more recent will work, or the master branch on GitHub as of 2021-12-07 or more recent. (TODO create that release before this blog post is published.)
```

Comment:
You'll sort out these todos before publishing? I guess we can publish this today, but up to you and Fin and Jacob if you will mention them of course.

Reply:
Yeah, for the TODO about mentioning them, I'd like them to tell me what they prefer.
For the release TODO, I'm waiting for their feedback to make a final pass and then I'll want to make a new release once we're happy with it all. I already have another commit to merge, so I don't want to release too soon...